pkg/lib/get-timesync-backend.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Ugh, how painful! (This is not a "no", just an "OMG why do we need to deal with such crap..)